_security/configuration/disable-enable-security.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: default
+title: Disabling and enabling the Security plugin
+parent: Configuration
+nav_order: 40
+has_toc: true
+redirect_from: 
+ - /security-plugin/configuration/disable/
+---
+
+# Disabling and enabling the Security plugin
+
+The Security plugin is installed by default when using OpenSearch, but you can temporarily disable it or remove it altogether. Disabling the plugin involves a change to the `opensearch.yml` file, and you may want to do this to streamline testing. A more substantive change is to remove the Security plugin completely. You might want to remove it if, for example, you have your own security solution or need to remove it for development purposes. 
+
+Disabling or removing the plugin exposes the configuration index for the Security plugin. If the index contains sensitive information, make sure to protect it through some other means. If you no longer need the index, delete it.
+{: .warning }
+
+Disabling, removing, or installing the Security plugin requires a full cluster restart to enable security features.
```

Comment:
I don't understand this line. Why would disabling or removing the plugin require a cluster restart to enable security features? Isn't this just on the installation side?

Reply:
There are changed to the opensearch.yml file, which needs a restart to take effect, the rest of the nodes will not be able to communicate with this node after restart, therefore full cluster restart is necessary